doc/GetFileProperties.md

```diff
@@ -0,0 +1,70 @@
+# GetOidForPath
```

Comment:
Isn't this supposed to be for `GetFileProperties`?

Reply:
Good catch, fixed